data/core/macros/special-notes.cfg

```diff
@@ -245,3 +245,6 @@ _"This unit has a defense cap on certain terrain types — it cannot achieve a h
 #define SPECIAL_NOTES_DEFENSE_CAP
 #deprecated 2 1.19 This note is now added automatically.
 {INTERNAL:SPECIAL_NOTES_DEFENSE_CAP}#enddef
+
+#define INTERNAL:SPECIAL_NOTES_PRECISE_STRIKE
+_"This unit’s great skill gives it a high chance of hitting targeted enemies, but only while attacking."#enddef
```

Comment:
would it make sense to add translation comment that it may be translated same as marksman for languages where current was already fitting?

Reply:
translation comment?

Reply:
In the line immediately above the translation string, you can write `#po: <explanatory text>` and it will be included as a comment in the `po` files that are used for translation.